src/browser/upload-file.js

```diff
@@ -27,7 +29,7 @@ function uploadFileBrowser (options = {}) {
         return reject(error)
       }
     }
-    xhr.send(options.file)
+    xhr.send(options.formData)
```

Comment:
got it. I did not find both are used anywhere. but `file` is not good name here because in real it is a `form data` with a single field `file` inside. Will replace back for now.